erizo/src/erizo/MediaStream.cpp

```diff
@@ -447,7 +451,7 @@ void MediaStream::initializePipeline() {
   pipeline_->addService(packet_buffer_);
 
   pipeline_->addFront(std::make_shared<PacketReader>(this));
-
+  addHandlerInPosition(BEGINNING, handler_pointer_dic, handler_order);
```

Comment:
I agree with the other reviewer :). I think we need better names for this. I propose a different way which would be more self-explanatory to me: `AFTER_READER`, `MIDDLE`, `BEFORE_WRITER`. Wdyt?

Reply:
Ok, great sugestion. I was also looking for a better name but couldn't find anything that clear.